samples/suit/smp_transfer/rad_envelope.yaml.jinja2.digest

```diff
@@ -0,0 +1 @@
+9d46b70df831d38071543b56193edaa269dfadbab4da8f384459a0a141a834a4
```

Comment:
If you need to put a dedicated template I would suggest putting them into `templates/recovery` without the digests.

Reply:
This was of course committed here by accident, removed.